dist/common/scripts/scylla-housekeeping

```diff
@@ -61,7 +61,10 @@ def sh_command(*args):
     return out
 
 def get_url(path):
-    return urllib.request.urlopen(path).read().decode('utf-8')
+    try:
+        return 0, urllib.request.urlopen(path).read().decode('utf-8')
+    except Exception as exc:
+        return 1, str(exc)
```

Comment:
Why are you returning the exception in such a strange way? I understand what you're doing, but not why you're using such a non-conventional way to do it.

Reply:
the whole issue that this code it running in a different process and the exception in some cases isn't serializable
hence why it's converted to string, and pass back with some kind of status code

Reply:
Please add a comment to explain it.

Reply:
Done